permissions/Compatibility.permission

```diff
@@ -0,0 +1,41 @@
+# -*- mode: sh -*-
+# This provides compatibility for older apps
+# The permission is not shown to user,
+# thus it ***must not*** give access to additional data or hardware resources
+
+# x-sailjail-translation-catalog =
+# x-sailjail-translation-key-description = permission-la-compatibility
+# x-sailjail-description = Compatibility
+# x-sailjail-translation-key-long-description = permission-la-compatibility_description
+# x-sailjail-long-description =
+
+# PDF tools
+private-bin pdfattach
+private-bin pdfdetach
+private-bin pdffonts
+private-bin pdfimages
+private-bin pdfinfo
+private-bin pdfseparate
+private-bin pdfsig
+private-bin pdftocairo
+private-bin pdftohtml
+private-bin pdftoppm
+private-bin pdftops
+private-bin pdftotext
+private-bin pdfunite
+
+# Calligra for office file conversion
+private-bin calligraconverter
```

Comment:
For calligraconverter however, i don't see any alternatives in the short term, or that i could do independently.
In the longer term, perhaps sailfish-office can grow a share-as-pdf functionality, or even more formal print functionality...